pkg/plugins/aws/ami/helpers.go

```diff
@@ -0,0 +1,65 @@
+package ami
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+	"github.com/sirupsen/logrus"
+)
+
+// GetLatestAmiID query the AWS API to return the newest AMI image id
+func (a *AMI) GetLatestAmiID(svc ec2iface.EC2API) (string, error) {
```

Comment:
```suggestion
func (a *AMI) getLatestAmiID(svc ec2iface.EC2API) (string, error) {
```
this method could be private

. Warning: you'll have to update the references across sources :)